src/transformState.js

```diff
@@ -4,8 +4,25 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  actions.forEach((el) => {
+    if (el.type === 'addProperties') {
+      Object.assign(state, el.extraData);
+    }
+
+    if (el.type === 'removeProperties') {
+      el.keysToRemove.forEach((elem) => {
+        if (Object.keys(state).includes(elem)) {
+          delete state[elem];
+        }
```

Comment:
Можно выполнять delete сразу, без проверки
```suggestion
        delete state[elem];
```